README.md

```diff
@@ -63,6 +63,19 @@ you we created a parser that is local. You can open it by copying the path /
 browser then upload the claim.json file of that run.
 ![overview](docs/assets/images/htmlpage.png)
 
+## Upload a previous feedback to the html parser page
+
+A feature on the HTML parser page that we did to allow users to upload a previous
```

Comment:
I think these two new paragraphs need to be reviewed and rephrased. My feeling is that this is a bit hard to read and understand right now, honestly.

Reply:
I agree.